bcda/models/models.go

```diff
@@ -166,20 +166,29 @@ type ACO struct {
 	ACOBeneficiaries []*ACOBeneficiary
 }
 
-func (aco *ACO) GetBeneficiaryIDs() (beneficiaryIDs []string, err error) {
+func (aco *ACO) GetBeneficiaryIDs() (cclfBeneficiaryIDs []string, err error) {
+	if aco.CMSID == nil {
+		log.Errorf("No CMSID set for ACO: %s", aco.UUID)
+		return cclfBeneficiaryIDs, fmt.Errorf("no CMS ID set for this ACO")
+	}
 	db := database.GetGORMDbConnection()
 	defer database.Close(db)
+	var cclfFile CCLFFile
+	// should I put a filter here to make sure it isn't too old?
```

Comment:
I don't know the answer to this question.  Should it become a `TODO` and a follow-up ticket in JIRA?

Reply:
If we decide the answer is yes then we should.  If we don;t let's leave it as is and when the ticket comes up we can do it.

Reply:
I like the idea of making this a `TODO` and creating a JIRA ticket. Otherwise, let's take the comment out.

Reply:
I talked to Sara and this needs to be a ticket so I will make one and update.